components/swatch/stories/template.js

```diff
@@ -72,7 +88,7 @@ export const Template = ({
 				updateArgs({ isSelected: !isSelected });
 			}}
 		>
-			${when((typeof imageUrl !== "undefined") && !isDisabled && !isMixedValue, () => html`
+			${when((typeof imageUrl !== "undefined") && !isDisabled && !isMixedValue && !isAddSwatch, () => html`
```

Comment:
Should we include a note in the changeset that you can't use an image with the add swatch but I think you can use a color right?